agora_analytics/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a752523e-9c3d-458c-a91a-af0c9fae5adc",
+  "app_id": "agora-analytics",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View Agora Analytics Collector data, such as freeze rate, in Metrics Explorer",
+    "title": "Agora Analytics",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Agora Analytics: Datadog Metrics Explorer Audio Freeze Rate",
+        "image_url": "images/agora_audio_freeze_rate.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Monitoring",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Mac OS",
+      "Supported OS::Windows"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "Agora Analytics",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "agora.",
+        "metadata_path": "metadata.csv"
```

Comment:
This is missing the metrics check field as mentioned [here](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file) under assets[integration[metrics[check]]]. Please add a metrics check

Reply:
added